hazelcast/discovery.py

```diff
@@ -1,15 +1,11 @@
 import json
 import logging
+import ssl
 
 from hazelcast.errors import HazelcastCertificationError
 from hazelcast.core import AddressHelper
 from hazelcast.six.moves import http_client
 
-try:
-    import ssl
-except ImportError:
```

Comment:
What was the reason for this check? `ssl` seems to be available in both Python 2 and 3.

Reply:
As you said, it was an unnecessary check

Reply:
Thanks for the confirmation.